forward/forward.go

```diff
@@ -169,13 +172,6 @@ func (f *Forwarder) cloneRequest(req *http.Request, u *url.URL) (*http.Request,
 	outReq.URL.Scheme = "http"
 	// We need to make sure the host is defined in the URL (not the actual URI)
 	outReq.URL.Host = req.Host
-	// Make sure we define an opaque URL, so the URI is just the path
-	//
-	// Note that EscapedPath() always return the URL-encoded path, so we have
-	// no way to know if the raw URL in req is encoded or not.  Go HTTP client
-	// always encode URL, so do major browsers. That means it's totally safe for
-	// Lantern usage and possibly others.
-	outReq.URL.Opaque = req.URL.EscapedPath()
```

Comment:
It looks like we've always populated the Opaque property, just with different values over time - https://github.com/getlantern/http-proxy/commit/72fd74b8153c91a10f53c933e5dd1286684d2658

My question is, why populate it at all?